kommunicateui/src/main/res/values/strings.xml

```diff
@@ -227,4 +227,21 @@
     <string name="km_active_bots_text">Active bot:</string>
     <string name="km_default_assigned_bot_name">Support Bot</string>
     <string name="km_take_over_from_bot_button_text">Take over from bot</string>
+    <string name="resolve_info_text1">later</string>
+    <string name="resolve_info_text2">la</string>
+    <string name="resolve_info_text3">te</string>
+    <string name="rating_status_text">r</string>
+    <string name="conversation_rated_info">You rated this conversation</string>
+    <string name="feedback_comment">Hey this is a test feedback and this is just a test. I will repeat this conversation again again again.</string>
```

Comment:
Where will this test feedback be displayed? Is it required in sdk?

Reply:
no its not, ive removed it